libs/designsystem/form-field/src/form-field.component.ts

```diff
@@ -264,4 +247,30 @@ export class FormFieldComponent
       !this.affixElements.some((affix) => affix.type === 'suffix') // there are no suffix elements
     );
   }
+
+  private handleAffixOffset() {
+    // Measure the width of all slotted affix elements,
+    // and apply their width + standard padding to the input elements
+    // padding, so the start/end of the input is correctly indented.
+    if (this.input) {
+      this.affixElements.forEach((affix) => {
+        this.resizeObserverService.observe(affix.el, (entry) => {
+          const padding = affix.type === 'prefix' ? 'padding-left' : 'padding-right';
+          const affixWidth = this.input.nativeElement.type === 'date' ? 0 : entry.contentRect.width;
```

Comment:
I think we can reduce this to
```suggestion
          const affixWidth = entry.contentRect.width;
```

Reply:
Prolly not a good idea Mr....

<img width="827" height="109" alt="Image" src="https://github.com/user-attachments/assets/4979caff-a3c1-4f3a-9327-d7a33e87ecd9" />